src/features/getData.js

```diff
@@ -10,3 +10,11 @@ export const getPopularData = async (dataName, page = 1) => {
 
   return data.results;
 };
+
+export const getGenreName = async () => {
```

Comment:
We're fetching genres not genre name
How about renaming it to getGenres or smth 😉